.travis.yml

```diff
@@ -12,6 +12,8 @@ cache:
 env:
   - DJANGO_SETTINGS_MODULE="tracpro.settings.tests"
 install:
+  - psql --version
+  - echo "select version();" | psql
```

Comment:
Did you want to keep these around? Fine if so... just checking to make sure it wasn't a temporary debugging step you wanted to do as a one-time.

Reply:
I might as well clean these up, though they don't hurt anything.